src/app/onboarding/styles.ts

```diff
@@ -0,0 +1,105 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import styled, { css } from 'styled-components';
+import { UnstyledButton } from '@/components/Buttons';
+import COLORS from '@/styles/colors';
+import { P } from '@/styles/text';
+
+export const OuterDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  min-height: 100vh;
+  padding-top: 40px;
+  padding-bottom: 100px;
+`;
+
+export const FormContainer = styled.div`
+  display: grid;
+  place-items: center;
+  flex-grow: 2;
+  width: 100%;
+`;
+
+export const FormFieldsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: min(528px, 100%);
+  margin: auto;
+  gap: 40px;
+`;
+
+export const LogoImage = styled(Image)<{ $show: boolean }>`
+  display: ${({ $show }) => ($show ? 'unset' : 'none')};
+  cursor: pointer;
+`;
+
+export const SectionBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  padding: 50px;
+  gap: 40px;
+  border: 1px solid ${COLORS.greyLight};
+  border-radius: 10px;
+`;
+
+export const SectionField = styled.div<{ $optional?: boolean }>`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  width: 100%;
+
+  & > h4 {
+    ${({ $optional }) =>
+      !$optional &&
+      `
+      &::after {
+        content: ' *';
+        color: ${COLORS.redMid};
+      }
+    `}
+  }
+`;
+
+const BackLinkStyles = css`
+  position: absolute;
+  top: 0;
```

Comment:
why not top: 30px; left 30px; instead of margin stuff?

Reply:
..... same thing....

Reply:
made it `top: 30px; left: 30px;`